api/v1/openapi.json

```diff
@@ -0,0 +1,1115 @@
+                                                        "$ref": "#/definitions/Error"
+                                                }
+                                        }
+				}
+			}
+		},
+                "/policy/resolve": {
+			"get": {
+				"summary": "Resolve policy for an identity context",
+				"tags": [ "policy" ],
+				"parameters": [
+					{ "$ref": "#/parameters/identity-context" }
+				],
+				"responses": {
+					"200": {
+                                                "description": "Success",
+                                                "schema": {
+							"$ref": "#/definitions/PolicyTraceResult"
+                                                }
+					}
+				}
+			}
+		},
+                "/service": {
+			"get": {
+				"summary": "Retrieve list of all services",
+				"tags": [ "service" ],
+				"responses": {
+					"200": {
+                                                "description": "Success",
+						"schema": {
+							"type": "array",
+							"items": {
+								"$ref": "#/definitions/Service"
+							}
+						}
+					}
+				}
+			},
+			"delete": {
+				"summary": "Delete all services",
+				"tags": [ "service" ],
+				"parameters": [
+					{
+						"name": "all",
+						"in": "query",
+						"type": "boolean"
+					}
+				],
+				"responses": {
+					"200": {
+                                                "description": "Success"
+					},
+                                        "default": {
+                                                "description": "Unexpected error",
+                                                "schema": {
+                                                        "$ref": "#/definitions/Error"
+                                                }
+                                        }
+				}
+			}
+		},
+                "/service/{id}": {
+			"get": {
+				"summary": "Retrieve configuration of a service",
+				"tags": [ "service" ],
+				"parameters": [
+					{ "$ref": "#/parameters/service-id" },
+					{ "$ref": "#/parameters/service-address" }
+				],
+				"responses": {
+					"200": {
+                                                "description": "Success",
+                                                "schema": {
+							"$ref": "#/definitions/Service"
+                                                }
+					},
+                                        "400": {
+                                                "description": "Address or ID parameter required"
+                                        },
+                                        "404": {
+                                                "description": "Service not found"
+                                        }
+				}
+			},
+			"put": {
+				"summary": "Create or update service",
+				"tags": [ "service" ],
+				"parameters": [
+					{ "$ref": "#/parameters/service-id" },
+					{ "$ref": "#/parameters/service-config" }
+				],
+				"responses": {
+					"201": {
```

Comment:
I've changed it to `200` for update successful and `201` for created to see whether create or update was performed.